.github/workflows/render.yml

```
@@ -6,18 +6,22 @@ jobs:
   render-peps:
     name: Render PEPs
     runs-on: ubuntu-latest
+    strategy:
+      fail-fast: false
+      matrix:
+        python-version: ["3.x", "3.11-dev"]
 
     steps:
       - name: 🛎️ Checkout
         uses: actions/checkout@v3
         with:
           fetch-depth: 0  # fetch all history so that last modified date-times are accurate
 
-      - name: 🐍 Set up Python 3
-        uses: actions/setup-python@v3
+      - name: Set up Python ${{ matrix.python-version }}
```

```suggestion
      - name: 🐍 Set up Python ${{ matrix.python-version }}
```

I'm not a huge personal emoji fan, but since we have them on all the other steps, it seems inconsistent to drop just this one, no?

---

Yes, unintentionally dropped via copy and paste from another file, will add back :)